src/js/dataMapping/award/awardAmountsSection.js

```diff
@@ -21,8 +21,16 @@ export const caresActSpendingCategories = [
 export const mockAwardIdsForCaresAct = [
     // contract
     'CONT_AWD_N0001917C0001_9700_-NONE-_-NONE-',
+    // exceeds current contract
```

Comment:
Please create a Jira task to remove the mocked data before launch!

Reply:
https://federal-spending-transparency.atlassian.net/browse/DEV-5434

Reply:
Thank you!